content/07.pathogenesis.md

```diff
@@ -46,22 +46,30 @@ One severe example includes reports of cytokine storm-like responses in patients
+A simple mechanistic model used to describe infectious disease dynamics is a susceptible-infected-recovered (SIR) state-transition model. 
+In this formulation individuals move through three states: susceptible, infected, and recovered; two parameters, $\gamma$ and $\beta$, are specified, the rate at which the infectious recover, and the infection transmission rate, respectively. 
+In this simple formulation, R~0~ is estimated as the ratio of $\beta$ and $\gamma$.[@doi:10.1126/science.abb5659; @isbn:9780691116174]. 
+A pathogen can invade a susceptible population if R~0~ > 1 [@isbn:9780691116174; @isbn:9780199209996].
+The spread of an infectious disease at a particular time t can be quantified by R~t~, the effective reproduction number, which assumes that part of the population may already be infected or that mitigating interventions were put into place. 
+For example, if only a fraction S~t~ of the population is still susceptible, R~t~ = S~t~ x R~0~.
 When R~t~ is greater than 1, an epidemic grows (i.e., the proportion of the population that is infectious increases); when R~t~ is less than 1, the proportion of the population that is infectious decreases.
 R~0~ and R~t~ can be estimated directly from epidemiological data or inferred using mathematical modeling.
-Modeling approaches are typically based upon a classic epidemiological model structure: the susceptible-infected-recovered (SIR) model and its extensions [@https://doi.org/10.1098/rspa.1927.0118; @doi:10.1038/280361a0].
+Modeling approaches are typically based upon a classic epidemiological model structure: the SIR model and its extensions [@https://doi.org/10.1098/rspa.1927.0118; @doi:10.1038/280361a0]. 
+In the context of SARS-CoV-2, more complex modified susceptible-exposed-infectious-recovered (SEIR) models are commonly used.[need citations]
 
 Estimates of R~0~ for COVID-19 lie in the range R~0~=1.4-6.5 [@doi:10/ggjvr7; @doi:10.1093/jtm/taaa021; @doi:10.1126/science.abb3221].
-Variation in R~0~ is expected between different populations and the estimated values of R~0~ discussed below are for specific populations in specific environments. The different estimates of R~0~ should not be interpreted as a range of estimates of the same underlying parameter.
+Variation in R~0~ is expected between different populations and the estimated values of R~0~ discussed below are for specific populations in specific environments. 
+The different estimates of R~0~ should not necessarily be interpreted as a range of estimates of the same underlying parameter.
+
+<!--I am not quite sure about this next paragraph. Most sophisticated models now incorporate "data" in their SEIR formulation, so I'm not sure it's accurate to draw the distinction.-->
+
 Data-derived estimates (i.e. those that do not incorporate SIR-type models into their analysis) typically predict lower values of R~0~.
```

Comment:
I agree wording is awkward since SEIR models use data. Suggestion below. also fine to just remove the sentence.
```suggestion
Estimates of R~0~ that relied only on data and did not use a SEIR model in their analyses typically predicted lower values.
```